src/services/github.service.ts

```diff
@@ -0,0 +1,16 @@
+import { GitHubRepo } from "../shared/models/github";
+
+class GitHubService {
```

Comment:
Is it necessary to implement service in Class? There is no problem with class in React, but since we are using hooks throughout all our projects, we need to stick with a function-based implementation maybe?

Reply:
Agree.